.gitignore

```
@@ -1,4 +1,5 @@
 dist
 node_modules
 .vscode
-src/notebook.ts
\ No newline at end of file
+src/notebook*.ts
+coverage
```

```suggestion
coverage
`.nyc_output`
```
Like the coverage the `.nyc_output` should be ignored.